### DISCUSSION TITLE
Change default time scale from seconds to minutes

### DISCUSSION THREAD

**Discussion by maxfield-allison** [jseidl/magic-areas#254] [Ideas / Feature Requests]
in many parts of the config flow "seconds" doesn't make the most sense for the time scale needed. for instance presence hold auto off is limited to 9999 seconds, roughly 2 and 3/4 hours. that said, polling rates should probably stay at seconds. I think we may have touched on this in a previous, closed issue.

**Comment by jseidl** [comment]
True, there are cases (as the presence hold timeout) where seconds is just dumb. But it's a breaking change, so major version work!

**Comment by jseidl** [comment]
You voted, it's coming out! https://github.com/jseidl/hass-magic_areas/pull/355

Next release coming soon

**Reply by maxfield-allison** [reply]
oh cool! haha forgot about this but happy to see you still plugging away at this project! the recent changes since last release have stabilized everything really well.

**Comment by jseidl** [comment]
Thanks for the feedback! Yeah now that my daughter is a bit older I'm being able to slowly catch up on the feature requests. Next update coming next week should have most of the most requested features such as calculated area light, floor meta areas and more!
